UI/cypress/e2e/shortlist.feature

```diff
@@ -200,12 +200,12 @@ Feature: Tuition Partner shortlist
     Given a user has selected TPs to shortlist and journeyed forward to the shortlist page
     Then the 'Tutors Green' price is 'From £10.42'
     When 'In School' tuition type shortlist refinement option is selected
-    Then 'Booster Club' is entry 1 on the shortlist page
-    And 'Zen Educate' is entry 2 on the shortlist page
-    And 'Tutors Green' is entry 3 on the shortlist page
-    And 'Reeson Education' is entry 4 on the shortlist page
-    And 'Action Tutoring' is entry 5 on the shortlist page
-    And '3D Recruit Ltd' is entry 6 on the shortlist page
+    Then 'Career Tree' is entry 1 on the shortlist page
```

Comment:
Please see my review comment above regarding keeping the test case ordering.

Reply:
As above - using actual TP data, not mocked up, so tests need to be updated.